### ISSUE TITLE
Doesn't work well with "aDetailer" extension.

### ISSUE THREAD

**Issue by GalaxyTimeMachine** [artyfacialintelagent/CloneCleaner#5]
If aDetailer is enabled, it tends to replace the face with a generic one, ignoring all settings from CloneCleaner. Would it be possible to pass the parameters to the prompt in aDetailer, so it keeps the unique faces?

**Comment by OliverTempel** [comment]
Can you check If you enable aDetailer, leave Prompt empty, so aDetailer gets the right prompt. i had the same issue, after leaving aDetailer prompt empty, it gets better

**Comment by GalaxyTimeMachine** [comment]
It does work better without a prompt in aDetailer, but then it is reducing the capabilities of that extension. Same with the issue with having Dynamic Prompts enabled, and all the CloneCleaner prompts are the same in every image of a batch.

**Comment by OliverTempel** [comment]
Yes i know, already opened an issue for that Problem with batches and identified a Problem withnthe Extension dynamic prompts
